citi-onboarding/Eventos-CITi
# Feat/create why we do it on client

## What I did

- Create the Why We Do It component
- Create the global css file 
- Add background-color and limit horizontal overflow
- Disable delete why we do it function

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout feat/create-why-we-do-it-on-client
   1. git pull origin feat/create-why-we-do-it-on-client
   1. cd server/
   1. node index.js

1. Open another terminal at the folder of this projects
  
1. In terminal
    1. cd client/
    1. npm run start

1. At your browser
   1. Check if the content is aligned as in [https://www.figma.com/file/2UJF2r572kDAU4snBxUMwx/citi-eventos?node-id=0%3A1](url)
   1. Try to resize the page 
   1. Try to swipe to the sides to check if there's no overflow
   1. Toggle device emulation after pressing ctrl +shift + i
   1.  Check width 375px
   1. Try to delete the file at [http://localhost:3001/admin/why-we-do-its](url)
  
PS.: The bubble will look cut in half but, as the other components join the page, it will be shown entirely

1. To make sure if the bubble will work properly
   1. Open the code
   1. Go to client>  src > style.css
   1. Set #root height to 400vh
   1. Check if the positions are corect
